src/python/foglamp/admin_api/README.rst

```diff
@@ -1,39 +1,86 @@
 Introduction
 ------------
-This code originated from work described at http://steelkiwi.com/blog/jwt-authorization-python-part-1-practise/
+This code originated from `JWT AUTHORIZATION IN PYTHON, PART 1 <http://steelkiwi.com/blog/jwt-authorization-python-part-1-practise>`_
+
 
 Starting the server
 -------------------
 The foglamp start script starts the server on port 8080. There is currently no https option.
 
 Authentication
 --------------
-POST to /api/auth/login with username: and password: in JSON format.
+POSTing to auth/login accepts user credentials and responds with a JSON document containing 'access' and 'refresh' tokens.
+
+An access token must be provided in the 'authorization' header for all other requests except auth/refresh-token.
 
-A JSON document is returned with two keys, refresh_token and access_token.
+Access tokens expire after 15 minutes. Refresh tokens are valid for up to 7 days.
 
-The access token should be provided in the 'authorization' header for all API calls except /api/auth/refresh_token. It expires after 15 minutes.
+When a token has expired, requests fail with status code 401. When a token has not been provided or if the token is invalid, requests fail with status code 400.
 
-POST the refresh token to /api/auth/refresh_token to get a new access token. The refresh token expires after 7 days.
+auth/login is an expensive operation. POSTing to auth/refresh-token is faster and also responds with a JSON document containing access_token (but no refresh_token). The refresh token must be provided in the 'authorization' header.
+
+Base URI
+--------
+/api
 
 Methods
 -------
-- whoami
+- POST auth/login
+
+  - There is currently only one user named 'user' with password 'password.'
+  - Request:
+  - .. code-block::
```

Comment:
`.. code-block::python`

otherwise it will give error for argument. else use `.. highlight::` where language arg is optional and defaults to python.